dace/sdfg/state.py

```diff
@@ -1226,6 +1214,45 @@ def add_reduce(
         self.add_node(result)
         return result
 
+    def add_pipeline(
+            self,
+            name,
+            ndrange,
+            init_size=0,
+            init_overlap=False,
+            drain_size=0,
+            drain_overlap=False,
+            schedule=dtypes.ScheduleType.FPGA_Device,
```

Comment:
Interesting default schedule, but I guess it's ok.

Reply:
I don't think we need it for anything else :-)